target_chains/ethereum/entropy_sdk/solidity/EntropyEvents.sol

```diff
@@ -15,6 +22,13 @@ interface EntropyEvents {
         bytes32 blockHash,
         bytes32 randomNumber
     );
+    event RevealedWithCallback(
+        EntropyStructs.Request request,
+        bytes32 userRandomNumber,
+        bytes32 providerRevelation,
+        bytes32 blockHash,
```

Comment:
sorry if my earlier comment was not clear enough. I think we should keep blockHash in the existing events, but we can remove it from here since this is a new type of event.